torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1885,6 +1918,25 @@ def _simplify_floor_div(self, expr):
             self.evaluate_expr(eq_expr)
         return self.simplify(expr)
 
+    def _add_guard(self, expr: "sympy.Expr") -> None:
+        stack = get_debugging_stack()
+        guard = ShapeGuard(expr, stack)
+        if os.environ.get("TORCHDYNAMO_PRINT_GUARDS", None) == "1":
```

Comment:
We shouldn't be repeatedly querying the environment every time we add a guard

Reply:
fixed